src/index.js

```diff
@@ -38,7 +38,7 @@ import ParentNewsFeed from './components/pages/ParentNewsFeed';
 import PaymentSuccess from './components/pages/ParentHome/PaymentSuccess';
 import Cart from './components/pages/ParentHome/Cart';
 import ParentFamilyHome from './components/pages/ParentFamily/ParentFamilyHome';
-
+import NavBar from './components/common/NavBar';
 const store = createStore(rootReducers, applyMiddleware(thunk));
 
 ReactDOM.render(
```

Comment:
There are a lot of recommendations from github to change, but something have to stay since someone else will work on it and know what they need to use since it's already imported. There's no big major issues, everything else looks good.